CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 in development
 --------------
 
+* Removed support for medium-strength ciphers from default nginx configuration (#3244)
```

Comment:
Minor thing for the future changelog changes - we tend to follow chronological ordering so the newest changes go to the end of changelog :)